### ISSUE TITLE
Tests fail due to incompability of pyFFTW and scipy>=1.4 

### ISSUE THREAD

**Issue by flixha** [eqcorrscan/EQcorrscan#368]
**Describe the bug**
All EQcorrscan tests that rely on pyfftw-functions currently fail, because the pyfftw-functions cause an error on importing "_fftpack" from scipy.fftpack for scipy>=1.4 (i.e., pyfftw and scipy 1.4 are incompatible).

**To Reproduce**
Run tests in EQcorrscan (0.4 or master version) with scipy>=1.4 installed.

**Additional context**
I am reporting this behavior to help anyone wondering about why their new install of EQcorrscan is not working. By installing scipy1.3.3, the problem is avoided. It appears that the pyFFTW-developers are about to publish a new release within a few days to a few weeks ( [pyFFTW/pyFFTW#279](https://github.com/pyFFTW/pyFFTW/issues/279). Until then, one could introduce requirement of scipy<1.4 for the tests to run, or speculate on pyFFTW being updated within days.

**Comment by calum-chamberlain** [comment]
Thanks for that spot @flixha! I doubt I will get round to updating the dependancies and re-releasing until after pyFFTW updates.

The scipy removal of fftpack is likely related to numpy removal of fftpack which has led to issues in obspy as well (e.g. [2434](https://github.com/obspy/obspy/issues/2434)).

**Comment by calum-chamberlain** [comment]
By the way: did you run into this installing from conda - I can pin the dependency there without a full release cycle so it could be simple enough to do?  With pip installs I would have to create a full new release.

**Comment by flixha** [comment]
Hi @calum-chamberlain , thanks for the quick reply. Yes, I ran into the problem when installing EQcorrscan 0.4 via conda (and from conda-forge).  Then I tried via pip git+git the master-version straight from github, with the same problem. So if it is a very simple fix to pin the dependency via conda that's great - otherwise I would absolutely count on the pyFFTW update.

Thanks for the link to the obspy problem to be aware of. It explains why conda only wanted to install obspy 1.1.0 and not 1.1.1. The problems certainly seem related, with the pyFFTW vs. scipy>=1.4 incompability having appeared only in the last 3-4 weeks.
